.github/workflows/test-integration.yml

```
@@ -0,0 +1,95 @@
+name: Integration Tests
+
+# See: https://docs.github.com/en/actions/reference/events-that-trigger-workflows
+on:
+  push:
+    paths-ignore:
+      - "__tests__/**"
+      - ".github/**"
+      - "!.github/workflows/test-integration.ya?ml"
+      - "**.md"
+      - ".gitignore"
+      - "LICENSE"
+  pull_request:
+    paths-ignore:
+      - "__tests__/**"
+      - ".github/**"
+      - "!.github/workflows/test-integration.ya?ml"
+      - "**.md"
+      - ".gitignore"
+      - "LICENSE"
```

```suggestion
    paths:
      - ".github/workflows/test-integration.ya?ml"
      - "lib/**"
      - "action.yml"
  pull_request:
    paths:
      - ".github/workflows/test-integration.ya?ml"
      - "lib/**"
      - "action.yml"
```

I believe these paths are the only ones relevant to the workflow so we can make the workflow more maintainable and efficient by configuring it to only run on changes to them.

I think the previous goofy path filters (which came from [the workflow this was based on](https://github.com/arduino/setup-task/blob/main/.github/workflows/test-integration.yml)) were the result of my lack of understanding of the project at the time I wrote the workflow. I simply excluded the paths I was certain were irrelevant and let the workflow run on a change to any other. That was a safe approach at that time, but also inefficient.